server/DataLouna/LounaAdmin/views.py

```diff
@@ -293,33 +451,330 @@ class getUser(APIView):
-            examples={
-                "application/json": {
-                    'getUser_Error': 'ID not found'
-
-                }
-            }
+            examples={"application/json": {"getUser_Error": "ID not found"}},
         ),
     }
-    get_user_config = openapi.Parameter(
-        'chat_id',
-        in_=openapi.IN_QUERY,
-        description='Получение пользователя по chat_id',
-        type=openapi.TYPE_STRING,
-    )
 
-    @swagger_auto_schema(manual_parameters=[get_user_config], responses=response_schema_dict)
+    @swagger_auto_schema(
+        manual_parameters=[
+            openapi.Parameter(
+                "chat_id",
+                in_=openapi.IN_QUERY,
+                description="Получение пользователя по chat_id",
+                type=openapi.TYPE_STRING,
+            ),
+            openapi.Parameter(
+                name="Authorization",
+                description="Authorization token",
+                required=True,
+                type=openapi.TYPE_STRING,
+                in_=openapi.IN_HEADER,
+            ),
+        ],
+        responses=response_schema_dict,
+    )
     def get(self, request):
-        chat_id = self.request.query_params.get('chat_id')
+        chat_id = self.request.query_params.get("chat_id")
         user = modelUser.objects.filter(chat_id=chat_id).values()
         try:
             user[0]
         except IndexError:
-            return Response({'getUser_Error': 'ID not found'}, status=status.HTTP_404_NOT_FOUND)
+            return Response({"getUser_Error": "ID not found"}, status=status.HTTP_404_NOT_FOUND)
         return Response(user)
+
+
+class getNoviceNewsTellers(APIView):
+    permission_classes = [IsAuthenticated]
+
+    def get(self, request):
+        id = request.GET.get("id")
+        if id:
+            novice_news_tellers = NoviceNewsTellers.objects.filter(id=id).values()
+            return Response(novice_news_tellers)
+        else:
+            novice_news_tellers = NoviceNewsTellers.objects.all().values()
+            return Response(novice_news_tellers)
+
+
+class getInactiveNewsTellers(APIView):
+    permission_classes = [IsAuthenticated]
+
+    def get(self, request):
+        id = request.GET.get("id")
+        if id:
+            inactive_news_tellers = InactiveNewsTellers.objects.filter(id=id).values()
+            return Response(inactive_news_tellers)
+        else:
+            inactive_news_tellers = InactiveNewsTellers.objects.all().values()
+            return Response(inactive_news_tellers)
+
+
+class onEnter(APIView):
+    permission_classes = [IsAuthenticated]
+    response_schema_dict = {
+        "200": openapi.Response(
+            description="200 Response",
+            examples={"application/json": {"Пользователь существует": "True"}},
+        ),
+        "400": openapi.Response(
+            description="400 Response",
+            examples={"application/json": {"Пользователь не существует": "False"}},
+        ),
```

Comment:
ключи в ответе сделать одинаковыми
```suggestion
        "200": openapi.Response(
            description="200 Response",
            examples={"application/json": {"exists": True}},
        ),
        "400": openapi.Response(
            description="404 Response",
            examples={"application/json": {"exists": False}},
        ),
```

Reply:
ещё коды поменять 400->404